draft-wussler-openpgp-pqc.md

```diff
@@ -660,8 +660,10 @@ row "ECC-KEM" of {{tab-ecdh-cfrg-artifacts}}, {{tab-ecdh-nist-artifacts}}, and
 The operations `x25519Kem.encap()` and `x448Kem.encap()` are defined as
 follows:
 
- 1. Generate an ephemeral key pair {`v`, `V=vG`}, without any masking or
-    clamping as defined in [RFC7748]
+ 1. Generate an ephemeral key pair {`v`, `V=vG`} according to
+    the procedure defined in RFC7748, that is, without any
+    measures such as masking or clamping that are applied
+    in previous OpenPGP specifications of this scheme.
```

Comment:
```suggestion
 1. Generate an ephemeral key pair {`v`, `V=vG`} according to
    the procedure defined in [RFC7748], that is, without any
    measures such as masking or clamping
```

Reply:
Probably it's better to remove the reference to older OpenPGP specs, but i like the sentence either way